bin/dfx-software-sns-aggregator-install.test

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+# Source the output file ----------------------------------------------------------
+source "$(clap.build)"
+
+# We make changes to dfx.json.  Better undo any changes.
+function cleanup {
+  git checkout dfx.json
+  dfx stop
+}
+trap cleanup EXIT
+
+for flavour in dev prod; do
+  for release in pinned latest proposal-115273-agg; do
+    (
+      echo "Should install the $release $flavour sns_aggregator"
+      : Clean up
+      git checkout dfx.json
+      dfx-network-stop
+      dfx start --clean --background 2>/dev/null
+      : Install
+      dfx-software-sns-aggregator-install --release "$release" --flavor "$flavour"
+      : The downloaded wasm should be the pinned, production version, not called 'pinned' or some mutable alias:
+      SNS_AGGREGATOR_RELEASE="$(. "$SOURCE_DIR/versions.bash" && echo "$SNS_AGGREGATOR_RELEASE")"
+      [[ "$SNS_AGGREGATOR_RELEASE" =~ ^proposal-[0-9]+-agg$ ]] || {
+        echo "ERROR: The pinned $release $flavour release should refer to a proposal: '$SNS_AGGREGATOR_RELEASE'"
```

Comment:
Why does it say `pinned` here? This code path is taken for each of the values or $release, right?

Reply:
Good catch.